<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_1">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>화남/흥분/신남</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>두려움/행복/평범</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>350</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Agency FB</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>슬픔/피곤/편안</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_driver">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_image">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>551</width>
     <height>431</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>lbl_image</zorder>
  <zorder>btn_1</zorder>
  <zorder>btn_2</zorder>
  <zorder>btn_3</zorder>
  <zorder>lbl_driver</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
